### PR TITLE
Close #36 

### LINKED ISSUE #36: ใส่ Indicator เพื่อบอกว่า ตั๋วรอบแรกขายหมดแล้ว
ตัวอย่าง:

image

### PR DESCRIPTION
**Pull request by our9planets**
เพิ่ม indicator soldout

### CODE REVIEW

**Comment by n3tr** [comment]
@our9planets Capture Screenshot มาใส่ในนี้หน่อยครับ :)

**Comment by our9planets** [comment]
screen shot 2017-05-18 at 6 49 55 pm

File: components/TicketsSection.js [hunk 1]
```diff
@@ -29,18 +29,31 @@ function TicketsTable () {
...
 
-function TicketsRow ({ title, date }) {
+function TicketsRow ({ title, date, soldout }) {
   return (
     <div className='tickets-row'>
-      <div className='title'>{title}</div>
-      <div className='date'>{date}</div>
+      { soldout && <span className='indicator-soldout' /> }
+      <div className={`title ${soldout && 'text-soldout'}`}>{title}</div>
+      <div className={`date ${soldout && 'text-soldout'}`}>{date}</div>
```
Comment (turboza) [inline components/TicketsSection.js]: @our9planets This way will generate the `undefined` class too na krub. 

credit @soraxtend - https://github.com/reactbkk/2.0.0/pull/43#discussion_r117229732

Comment (our9planets) [reply components/TicketsSection.js]: Thank you

**Comment by dtinth** [comment]
Sorry, I have to merge another PR, #43, instead as that PR finished first.

Thanks for your participation. There is still a lot to improve in this web site, so please don’t hesitate if you have anything to contribute to it 😄

**Pull request closed without merging**